modules/thermal_hydraulics/doc/content/modules/thermal_hydraulics/syntax/ControlLogic/index.md

```diff
@@ -1 +1,25 @@
-!template load file=stubs/moose_system.md.template name=ControlLogic syntax=/ControlLogic
+# ControlLogic System
+
+The `ControlLogic` system is an extension of MOOSE's [Controls system](syntax/Controls/index.md).
+Standard MOOSE `Control`s can be created with the `ControlLogic` syntax.
+`THMControl`s, however, which have additional functionality, require
+the `ControlLogic` syntax.
```

Comment:
this read weird. 
Do you really mean it s just an option in MOOSE and mandatory in THM

Reply:
I'm trying to say that you can do this
```
[ControlLogic]
  [some_moose_control]
    ...
  []
  [some_thm_control]
    ...
  []
[]
```
but not this:
```
[Controls]
  [some_moose_control]
    ...
  []
  [some_thm_control] # this is an error
    ...
  []
[]
```